ceph_iscsi_config/settings.py

```diff
@@ -51,7 +51,41 @@ class Settings(object):
         "max_data_area_mb": {
             "min": 1, "max": 2048},
         "osd_op_timeout": {
-            "min": 0, "max": 600}
+            "min": 0, "max": 600},
+
+        "emulate_3pc": {
+            "min": 0, "max": 1},
+        "emulate_caw": {
+            "min": 0, "max": 1},
+        "emulate_dpo": {
+            "min": 0, "max": 1},
+        "emulate_fua_read": {
+            "min": 0, "max": 1},
+        "emulate_fua_write": {
+            "min": 0, "max": 1},
+        "emulate_model_alias": {
+            "min": 0, "max": 1},
+        "emulate_pr": {
+            "min": 0, "max": 1},
+        "emulate_rest_reord": {
```

Comment:
Restricted reordering is not implemented. Kernel LIO will throw an error if ```emulate_rest_reord``` is set to boolean true (1/y/Y).